packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -7071,3 +7071,62 @@ class _StatefulBuilderState extends State<StatefulBuilder> {
   @override
   Widget build(BuildContext context) => widget.builder(context, setState);
 }
+
+/// A widget that paints its area with a specified [Color] and then draws its
+/// child.
```

Comment:
maybe be specific and say that the child is drawn on top of that color?

Reply:
(even though that's kinda obvious, I guess..)

Reply:
Done